gui/directoryservice/migrations/0011_add_new_idmap_model.py

```diff
@@ -0,0 +1,259 @@
+        migrations.AddField(
+            model_name='idmap_nss',
+            name='idmap_nss_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_rfc2307',
+            name='idmap_rfc2307_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_rid',
+            name='idmap_rid_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_script',
+            name='idmap_script_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_tdb',
+            name='idmap_tdb_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_tdb',
+            name='idmap_tdb_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.AddField(
+            model_name='idmap_tdb2',
+            name='idmap_tdb2_domain',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, 
+                to='directoryservice.idmap_domain', 
+                to_field='idmap_domain_name', 
+                null=True, verbose_name='pre-Windows 2000 Domain Name'
+            ),
+        ), 
+        migrations.RunPython(migrate_to_new_idmap, reverse_code=revert_idmap_changes),       
```

Comment:
You can use `migrations.RunPython.noop` instead of an empty method.

Reply:
Good to know. Thanks!